pkg/cmd/util/clientcmd/factory.go

```diff
@@ -323,7 +323,7 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			var err error
 			var pods *api.PodList
 			for pods == nil || len(pods.Items) == 0 {
-				if t.LatestVersion == 0 {
+				if t.Status.LatestVersion == 0 {
 					time.Sleep(2 * time.Second)
```

Comment:
> #6337

Can we get a separate issue on a fixed `Sleep`.  Seems like a watch or even a Poll on a tighter interval would be nicer for the user.